a10_octavia/controller/worker/controller_worker.py

```diff
@@ -312,10 +354,42 @@ def create_load_balancer(self, load_balancer_id):
         store[constants.UPDATE_DICT] = {
             constants.LOADBALANCER_TOPOLOGY: topology
         }
+        validation_flag=False
+        flag=False
+        if self.config.has_option("RACK_VTHUNDER","devices") and self.config.has_section("RACK_VTHUNDER"):
```

Comment:
All this logic below which involves parsing the config, should be done in A10Config.